### PR TITLE
Add Russian translation for command line interface parameters

### PR DESCRIPTION
**Pull request by lifefire1**
See [JENKINS-XXXXX](https://issues.jenkins.io/browse/JENKINS-XXXXX).

### Proposed changelog entries

- Add Russian translation for command line interface parameters.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] The Jira issue, if it exists, is well-described.
- [ ] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)).
  - Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [ ] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7677"></a>

### CODE REVIEW

**Comment by lifefire1** [comment]
Screenshot_20230227_213143
this is an example output in bulgarian i made

**Comment by NotMyFault** [comment]
> Do I need to add the src/main/java/hudson/cli/client/Messages.java file, in which I will have output methods implemented

No, `src/main/java/hudson/cli/client/Messages.java` is generated at compile time and does not exist as a source file. 
You just need to modify the .properties file if you want to contribute a localization, in this case.

**Comment by NotMyFault** [comment]
@yaroslavafenkin or @Pldi23, would you mind taking a look at the Russian translation proposed if your schedule allows it? You have reviewed a few PRs like these before, hence I'm asking :)

**Comment by yaroslavafenkin** [comment]
> @yaroslavafenkin or @Pldi23, would you mind taking a look at the Russian translation proposed if your schedule allows it? You have reviewed a few PRs like these before, hence I'm asking :)

Will have a look when time permits

File: cli/src/main/resources/hudson/cli/client/Messages_ru.properties [hunk 1]
```diff
@@ -0,0 +1,26 @@
+CLI.Usage=Jenkins CLI\n\
+  Применение: java -jar jenkins-cli.jar [-s URL] command [opts...] args...\n\
+  Параметры:\n\
+  \ -s URL              : URL-адрес сервера (по умолчанию используется переменная env JENKINS-URL)\n\
+  \ -webSocket          : Подключится через WebSocket (по умолчанию; хорошо работает с большинством обратных прокси; требуется Jetty)\n\
```
Comment (Pldi23) [inline cli/src/main/resources/hudson/cli/client/Messages_ru.properties:5]:
```suggestion
  \ -webSocket          : Использовать WebSocket соединение (по умолчанию; хорошо работает с большинством обратных прокси; требуется Jetty)\n\
```

File: cli/src/main/resources/hudson/cli/client/Messages_ru.properties [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  \ -http               : Используется соединение HTTP(S), а не WebSocket\n\
```
Comment (Pldi23) [inline cli/src/main/resources/hudson/cli/client/Messages_ru.properties:6]:
```suggestion
  \ -http               : Использовать соединение HTTP(S), а не WebSocket\n\
```

File: cli/src/main/resources/hudson/cli/client/Messages_ru.properties [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  \ -ssh                : Использовать протокол SSH, а не WebSocket(требуется -user; порт SSH должен быть открыт на сервере)\n\
+  \ -i KEY              : Файл закрытого ключа SSH, используемый для аутентификации(для использования с -ssh)\n\
+  \ -noCertificateCheck : Полностью обойти проверку сертификата HTTPS. Используйте с осторожностью\n\
```
Comment (Pldi23) [inline cli/src/main/resources/hudson/cli/client/Messages_ru.properties:9]:
```suggestion
  \ -noCertificateCheck : Полностью отказаться от проверки сертификата HTTPS. Используйте с осторожностью\n\
```

File: cli/src/main/resources/hudson/cli/client/Messages_ru.properties [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  \ -noKeyAuth          : Не пытаться загрузить приватный ключ для SSH. Конфликты с -i\n\
+  \ -user               : Указать пользователя (для использования с -ssh; должен быть зарегистрирован открытый ключ)\n\
+  \ -strictHostKey      : Запросить строгую проверку ключа хоста (для использования с -ssh)\n\
```
Comment (Pldi23) [inline cli/src/main/resources/hudson/cli/client/Messages_ru.properties:12]:
```suggestion
  \ -strictHostKey      : Использовать строгую проверку ключа хоста (для использования с -ssh)\n\
```

**Review by Pldi23 — CHANGES_REQUESTED** [review]

File: cli/src/main/resources/hudson/cli/client/Messages_ru.properties [hunk 5]
```diff
@@ -0,0 +1,26 @@
+CLI.Usage=Jenkins CLI\n\
+  Применение: java -jar jenkins-cli.jar [-s URL] command [opts...] args...\n\
+  Параметры:\n\
+  \ -s URL              : URL-адрес сервера (по умолчанию используется переменная env JENKINS-URL)\n\
```
Comment (yaroslavafenkin) [inline cli/src/main/resources/hudson/cli/client/Messages_ru.properties:4]:
```suggestion
  \ -s URL              : URL-адрес сервера (по умолчанию используется переменная окружения JENKINS-URL)\n\
```

File: cli/src/main/resources/hudson/cli/client/Messages_ru.properties (hunk 2 above)
Comment (yaroslavafenkin) [inline cli/src/main/resources/hudson/cli/client/Messages_ru.properties:6]: Why are these two swapped? In English version `http` goes first, then `websocket`, would be nice to have consistent order.

Comment (yaroslavafenkin) [inline cli/src/main/resources/hudson/cli/client/Messages_ru.properties:6]: English version hints me it's the default connection option, maybe it should be mentioned?

File: cli/src/main/resources/hudson/cli/client/Messages_ru.properties [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+  \ -webSocket          : Подключится через WebSocket (по умолчанию; хорошо работает с большинством обратных прокси; требуется Jetty)\n\
+  \ -http               : Используется соединение HTTP(S), а не WebSocket\n\
+  \ -ssh                : Использовать протокол SSH, а не WebSocket(требуется -user; порт SSH должен быть открыт на сервере)\n\
+  \ -i KEY              : Файл закрытого ключа SSH, используемый для аутентификации(для использования с -ssh)\n\
+  \ -noCertificateCheck : Полностью обойти проверку сертификата HTTPS. Используйте с осторожностью\n\
+  \ -noKeyAuth          : Не пытаться загрузить приватный ключ для SSH. Конфликты с -i\n\
```
Comment (yaroslavafenkin) [inline cli/src/main/resources/hudson/cli/client/Messages_ru.properties:10]:
```suggestion
  \ -noKeyAuth          : Не пытаться загрузить закрытый ключ для SSH. Не используйте с -i\n\
```

File: cli/src/main/resources/hudson/cli/client/Messages_ru.properties [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+  \ -auth [ USER:SECRET | @FILE ] : укажите имя пользователя и пароль или укажите API token (или загрузите их оба из файла);\n\
+  \                                 для использования с -http.\n\
+  \                                 Рекомендуется передавать учетные данные файлом .\n\
+  \                                 Смотрите https://www.jenkins.io/redirect/cli-http-connection-mode для получения дополнительной информации и параметров.\n\
+  \ -bearer [ TOKEN | @FILE ]     : указать аутентификацию с использованием токена носителя (или загрузить токен из файла);\n\
+  \                                 для использования с -http. Взаимоисключающие с -auth.\n\
+  \                                 Рекомендуется передавать учетные данные файлом.\n\
+  \n\
+  Доступные команды зависят от сервера. Запустите команду ''help'' в \
+  смотреть список.
```
Comment (yaroslavafenkin) [inline cli/src/main/resources/hudson/cli/client/Messages_ru.properties:23]:
```suggestion
  Доступные команды зависят от сервера. Запустите команду ''help'' чтобы \
  посмотреть весь список.
```

File: cli/src/main/resources/hudson/cli/client/Messages_ru.properties [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+CLI.NoURL=Ни -s, ни переменная окружения JENKINS_URL не указаны..
+CLI.NoSuchFileExists=Такого файла не существует: {0}
+CLI.BadAuth=Обе переменные env JENKINS_USER_ID и JENKINS_API_TOKEN должны быть установлены или оставлены пустыми.
```
Comment (yaroslavafenkin) [inline cli/src/main/resources/hudson/cli/client/Messages_ru.properties:26]:
```suggestion
CLI.BadAuth=Обе переменные окружения JENKINS_USER_ID и JENKINS_API_TOKEN должны быть установлены или оставлены пустыми.
```

**Comment by NotMyFault** [comment]
Hey @lifefire1,

could you take a moment and address the feedback, please?

Thanks!

**Comment by NotMyFault** [comment]
I'm going ahead and closing the PR, due to lack of response from the submitter. 
@lifefire1 you are much welcome to reopen the PR, once you have addressed the feedback submitted, thanks!

**Pull request closed without merging**